app/src/main/java/net/bible/service/readingplan/ReadingPlanDao.kt

```diff
@@ -65,6 +68,25 @@ class ReadingPlanDao {
 
         }
 
+    val sdReadingPlanList: List<ReadingPlanInfoDto>?
```

Comment:
sd? Modern Android device does not necessarily have SD card, so probably we shouldn't use it any more in our terminology (although for legacy reasons it has been used in the codebase).

What is the correct term then? User storage? (vs. app storage, which user can't access but app can) Not sure. You could check this from documentation.

Reply:
No longer applies. Class deleted